grafana/overlays/moc/smaug/blackbox-exporter/service-monitor.yaml

```diff
@@ -21,6 +21,72 @@ spec:
           - 'https://jupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud/'
       path: /probe
       targetPort: 9115
+    - interval: 30s
+      metricRelabelings:
+        - sourceLabels:
+            - __address__
+          targetLabel: __param_target
+        - sourceLabels:
+            - __param_target
+          targetLabel: instance
+        - replacement: 'https://www.operate-first.cloud/'
+          targetLabel: target
+      params:
+        module:
+          - http_2xx
+        target:
+          - 'https://www.operate-first.cloud/'
+      path: /probe
+    - interval: 30s
+      metricRelabelings:
+        - sourceLabels:
+            - __address__
+          targetLabel: __param_target
+        - sourceLabels:
+            - __param_target
+          targetLabel: instance
+        - replacement: 'https://thanos-query-frontend-opf-observatorium.apps.smaug.na.operate-first.cloud/'
+          targetLabel: target
+      params:
+        module:
+          - http_2xx
+        target:
+          - 'https://thanos-query-frontend-opf-observatorium.apps.smaug.na.operate-first.cloud/'
+      path: /probe
+      targetPort: 10902
```

Comment:
The path and targetPort here should be those of the Blackbox Exporter. So they should be same for all endpoints.